crates/api/src/agent.rs

```diff
@@ -326,13 +326,14 @@ impl AgentInfoSigned {
         let v: Vec<Ref> = serde_json::from_slice(encoded)
             .map_err(|e| K2Error::other_src("decoding agent_info", e))?;
         Ok(v.into_iter()
-            .map(|v| {
-                Self::inner_decode_one(verifier, v.agent_info, v.signature)
-            })
+            .map(|v| Self::from_parts(verifier, v.agent_info, v.signature))
             .collect())
     }
 
-    fn inner_decode_one<V: Verifier>(
+    /// Create an instance from its parts.
+    ///
+    /// Useful when receiving an agent info over the wire.
+    pub fn from_parts<V: Verifier>(
```

Comment:
Exposed this so that I can reconstruct these after sending them over the network. Happy to be told there's a better way

Reply:
I was imagining you'd use the `encode` below to send a single string for agent info across the wire, then use the above decode to decode it.

Reply:
Done -> https://github.com/holochain/kitsune2/pull/72/commits/52be4176fd0491287dc47c50f0fd45076907078d